helm/cas-postgres/values.yaml

```diff
@@ -0,0 +1,32 @@
+patroni:
+  image:
+    repository:  gcr.io/ggl-cas-storage/cas-postgres
+    pullPolicy: Always
+    tag: "0.1.0"
+
+  replicaCount: 2
+
+  resources: {}
+  credentials:
+    random: true
+  persistentVolume:
+    storageClass: gluster-block
+    size: 1Gi
+  # As per https://patroni.readthedocs.io/en/latest/kubernetes.html#use-configmaps
+  # "in some cases, for instance, when running on OpenShift, there is no alternative to using ConfigMaps."
```

Comment:
Would love to know why Platform Services are using `etcd` and since we don't have that answer I'm leaning towards "fewer moving parts are better" and really like this approach.

Reply:
we know that if we loose consistency in k8s api then our whole app is cooked, whereas if we loose consistency in our own etcd instance that could happen because our etcd pods were killed